external/PATCH-01-STRATUM

```diff
@@ -694,9 +694,22 @@ index f553054d..3f41e9e1 100644
  }
 
 diff --git a/stratum/hal/lib/common/common.proto b/stratum/hal/lib/common/common.proto
-index f7a72d70..45736661 100644
+index f7a72d70..7d5f2fd6 100644
 --- a/stratum/hal/lib/common/common.proto
 +++ b/stratum/hal/lib/common/common.proto
+@@ -46,6 +46,12 @@ enum Platform {
```

Comment:
why is this needed? Did you add this?

Reply:
This is just a reference for patch file where to add new enum structure SWBackendQemuHotplugStatus. Only the lines with '++' is new code. Others are just reference line used when applying the patch